nautobot/extras/templatetags/job_buttons.py

```diff
@@ -69,115 +70,108 @@
 </div>
 """
 
+SAFE_EMPTY = mark_safe("")  # noqa: S308
```

Comment:
Same comment as in #4993, I think `SAFE_EMPTY_STR` is a more descriptive name for this variable.